test/functional/test_framework/test_framework.py

```diff
@@ -432,6 +453,7 @@ def _initialize_chain_clean(self):
         for i in range(self.num_nodes):
             initialize_datadir(self.options.tmpdir, i)
 
+
```

Comment:
please remove stray newline

Reply:
That was intentional from our conversation in #12884. This would follow PEP8 so I figured bundle in here. OK to remove still if you want